packages/ui/v2/form/fields.tsx

```diff
@@ -107,6 +107,16 @@ function HintsOrErrors<T extends FieldValues = FieldValues>(props: {
     );
   }
 
+  // errors exist, not custom ones, just show them as is
+  if (fieldErrors) {
+    return (
+      <div className="text-gray mt-2 flex items-center text-sm text-red-700">
+        <Info className="mr-1 h-3 w-3" />
+        {methods.formState.errors[fieldName].message}
```

Comment:
Wouldn't this work?

```suggestion
        {fieldErrors.message}
```

Reply:
Yes, thank you. I copied the non-refactored code that got deleted in the last refactor. Applied!